addons/wowl/doc/dropdown_component.md

```diff
@@ -0,0 +1,186 @@
+# Dropdown Component
+
+## Overview
+
+As dropdowns are common in Odoo, we decided to make a generic dropdown component.
+
+It contains all the logic of what you can usually expect a dropdown to behave.
+
+We made the choice to develop this simple component with as few style as possible, therefore it is your responsibility to make yours look awesome 😎!
+
+## Behind the scene
+
+A `<Dropdown/>` component is simply a `<div class="o_dropdown"/>` having a `<button/>` next to an unordered list (`<ul/>`). The button controls if the list is attached to the DOM or not.
```

Comment:
I'd say 'is present in the DOM' or simply 'is in the DOM' (we used to manually attach/detach stuff into/from the DOM, hopefully we no longer need to do that)